sdk/keyvault/Microsoft.Azure.KeyVault/tests/KeyVaultOperationsTest.cs

```diff
@@ -20,6 +20,10 @@
 using Microsoft.Rest.Serialization;
 using Xunit;
 
+// Avoid compiling the test class. Disabling just the test methods still causes the fixture to instantiate and dispose, which still breaks live tests.
+// See https://github.com/Azure/azure-sdk-for-net/pull/8423 for status and more information about why these tests are disabled.
```

Comment:
I would reference the issue here not the PR

Reply:
Oops. Definitely intended the issue. I'll fix that.